src/query/expression/src/values.rs

```diff
@@ -396,7 +407,7 @@ impl Scalar {
                 Scalar::Bitmap(buf)
             }
             DataType::Tuple(tys) => Scalar::Tuple(tys.iter().map(Scalar::default_value).collect()),
-            DataType::Variant => Scalar::Variant(vec![]),
+            DataType::Variant | DataType::Geometry => Scalar::Variant(vec![]),
```

Comment:
the default value of Geometry is not correct.

Reply:
Default value of geomety is more complex than other types.
 
![4FFE7F9D-26AD-40FC-ACFF-BEE157D4AE0A](https://github.com/datafuselabs/databend/assets/1751500/04120526-1b4e-45ef-a6fc-fa96637e67d8)

Reply:
If we don't know the inner type of geomety, it's hard to have default value (maybe default point value by default)